domapping/cli.py

```diff
@@ -81,14 +89,30 @@ def schema_to_mapping_cli(schema, output, config, indent):
 @click.argument('output', type=click.File('w'), default='-')
 @click.option('--indent', '-i', default=4, type=click.INT,
               help='Output template indentation step.')
-@click.option('--type', '-t', default='type', type=click.STRING,
+@click.option('--es_type', '-t', type=click.STRING,
               help='Root type name used in jinja block names.')
-def mapping_to_jinja_cli(mapping, output, indent, type):
+def mapping_to_jinja_cli(mapping, output, indent, es_type):
     """Generate jinja template from Elasticsearch mapping."""
+    default_type = 'type'
     parsed_mapping = json.load(mapping)
 
-    template = mapping_to_jinja(parsed_mapping, type, indent=indent)
-    output.write(template)
+    if 'mappings' in parsed_mapping:
+        n_mappings = len(parsed_mapping['mappings'])
+        assert not (n_mappings > 1 and es_type)
+
+        result = []
+        for item in parsed_mapping['mappings']:
+            template = mapping_to_jinja(
+                    parsed_mapping['mappings'][item],
+                    item if n_mappings != 1 or not es_type else es_type,
+                    indent=indent)
+            result.append('"{0}":{1}'.format(item, template))
```

Comment:
Maybe we could add the first indentation level so that it is formatted properly as the rest of the jinja template? I think that this should do the trick:
```'{0}"{1}":{2}'.format(indent*' ', item, template)```